### ISSUE TITLE
Some DB errors (last rev DB with all updates) (BB #1079)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1079]
This issue was migrated from bitbucket.
**Original Reporter:** PadreWoW
**Original Date:** 27.07.2015 07:24:20 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** closed
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079

<hr>

```
#!c++

2015-07-27 10:19:57 SourceEntry 33633 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 46589 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 46589 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)
2015-07-27 10:19:57 SourceEntry 37434 in `condition` table does not have any implicit target TARGET_UNIT_NEARBY_ENTRY(38) or TARGET_DST_NEARBY_ENTRY (46)                                ,TARGET_UNIT_AREA_ENTRY_SRC(7), TARGET_UNIT_AREA_ENTRY_DST(8), TARGET_UNIT_CONE_ENTRY(60), TARGET_GAMEOBJECT_NEARBY_ENTRY(40)

2015-07-27 10:19:57 Table spell_scripts has unsupported spell (Id: 35372) without SPELL_EFFECT_SCRIPT_EFFECT (77) spell effect

2015-07-27 10:19:56 Quest 10836 has ReqCreatureOrGOId1 = 15 but creature with entry 15 does not exist, quest cannot be completed.
2015-07-27 10:19:56 Spell (id: 33824) has SPELL_EFFECT_QUEST_COMPLETE for quest 10162 , but quest does not have flag QUEST_SPECIAL_FLAGS_EXPLORATION_OR_EVENT. Quest flags must be fixed, quest modified to enable objective.
2015-07-27 10:19:56 Spell 36563 (Shadowstep) misses spell_affect for effect 1
2015-07-27 10:19:56 Spell 36563 (Shadowstep) misses spell_affect for effect 2

2015-07-27 10:19:56 `pool_gameobject` has a not lootable gameobject spawn (GUID: 5512, type: 0) defined for pool id (4993), skipped.
2015-07-27 10:19:56 `pool_gameobject` has a not lootable gameobject spawn (GUID: 5518, type: 0) defined for pool id (4993), skipped.

2015-07-27 10:19:54 Spell 13549 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13550 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13551 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13552 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13553 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13554 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 13555 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 14279 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 14280 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 24132 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 24133 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 24134 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 24135 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 25295 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 27016 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 27018 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 27068 listed in `spell_group` is not first rank of spell
2015-07-27 10:19:54 Spell 27069 listed in `spell_group` is not first rank of spell
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 27.07.2015 07:28:56 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079/#comment-20483726

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 27.07.2015 07:30:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079/#comment-20483751

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 28.07.2015 09:28:54 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079/#comment-20516017

<hr>

Here is my small fix for some db errors (last rev):

Please, check it, and add to repository if it's good.

```
#!c++
UPDATE `world`.`creature_template` SET `ScriptName` = 'npc_phase_hunter' WHERE `entry` = '20906';
UPDATE `world`.`creature_template` SET `ScriptName` = 'npc_azaloth' WHERE `entry` = '21506';
UPDATE `world`.`creature_template` SET `ScriptName` = 'npc_sunfurywarlock' WHERE `entry` = '21503';

UPDATE `world`.`creature_template` SET `AIName` = 'EventAI' WHERE `entry` = '22523';
UPDATE `world`.`creature_template` SET `AIName` = 'EventAI' WHERE `entry` = '25213';
UPDATE `world`.`creature_template` SET `AIName` = '' WHERE `entry` = '17211'; 
UPDATE `world`.`creature_template` SET `AIName` = '' WHERE `entry` = '2164';
DELETE FROM `world`.`creature_ai_scripts` WHERE `creature_id` = '17211'; 
DELETE FROM `world`.`creature_ai_scripts` WHERE `creature_id` = '20906';
DELETE FROM `world`.`creature_ai_scripts` WHERE `creature_id` = '2164';

DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005006'; 
DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005007'; 
DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005008'; 
DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005009'; 
DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005010'; 
DELETE FROM `world`.`db_script_string` WHERE `entry` = '2000005017';

DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '21478' AND `Item` = '31372';
UPDATE `world`.`creature_loot_template` SET `Chance` = '100' WHERE `Entry` = '21477' AND `Item` = '31372'; 
UPDATE `world`.`creature_loot_template` SET `Chance` = '100' WHERE `Entry` = '4723' AND `Item` = '5852';
UPDATE `world`.`creature_loot_template` SET `QuestRequired` = '1' WHERE `Entry` = '21477' AND `Item` = '31372';

DELETE FROM `world`.`npc_gossip` WHERE `npc_guid` = '83989';

UPDATE `world`.`creature_template` SET `lootid` = '25593' WHERE `entry` = '25507';
INSERT INTO `world`.`reference_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES ('34092', '35205', '0', '0', '0', '1', '1', '1', NULL); 
INSERT INTO `world`.`reference_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES ('34092', '35214', '0', '0', '0', '1', '1', '1', NULL); 

DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25363';
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25367'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25368'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25369'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25370'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25371'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25483'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25486'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25507'; 
DELETE FROM `world`.`creature_loot_template` WHERE `Entry` = '25597';

UPDATE `world`.`quest_template` SET `ReqCreatureOrGOId1` = '22197' WHERE `entry` = '10836';

DELETE FROM `world`.`pool_gameobject` WHERE `guid` = '5512' AND `pool_entry` = '4993'; 
DELETE FROM `world`.`pool_gameobject` WHERE `guid` = '5518' AND `pool_entry` = '4993';

DELETE FROM `world`.`creature_addon` WHERE `guid` = '84009';

UPDATE `world`.`creature_addon` SET `auras` = '35194 0' WHERE `guid` = '64246';
UPDATE `world`.`creature_addon` SET `auras` = '5301 0' WHERE `guid` = '67906'; 
UPDATE `world`.`creature_addon` SET `auras` = '5301 0' WHERE `guid` = '67907'; 
UPDATE `world`.`creature_addon` SET `auras` = '5301 0' WHERE `guid` = '67910'; 

UPDATE `world`.`creature_template_addon` SET `auras` = '31261 0' WHERE `entry` = '21304'; 
UPDATE `world`.`creature_template_addon` SET `auras` = '' WHERE `entry` = '20869';
UPDATE `world`.`creature_template_addon` SET `auras` = '35194 0' WHERE `entry` = '18121'; 

DELETE FROM `world`.`creature_addon` WHERE `guid` = '11657';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '11693';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '11747';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14543';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14544';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14547';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14551';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14558';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14561';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14579';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14634';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '14638';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '45214';
DELETE FROM `world`.`creature_addon` WHERE `guid` = '45215';

UPDATE `world`.`creature_template` SET `heroic_entry` = '23181' WHERE `entry` = '23175'; 
UPDATE `world`.`creature_template` SET `heroic_entry` = '23182' WHERE `entry` = '23176'; 
UPDATE `world`.`creature_template` SET `heroic_entry` = '23183' WHERE `entry` = '23177'; 
UPDATE `world`.`creature_template` SET `heroic_entry` = '23184' WHERE `entry` = '23178'; 
UPDATE `world`.`creature_template` SET `heroic_entry` = '23185' WHERE `entry` = '23179'; 
UPDATE `world`.`creature_template` SET `heroic_entry` = '23186' WHERE `entry` = '23180';

DELETE FROM `world`.`creature_equip_template` WHERE `entry` = '3005';
UPDATE `world`.`creature_template` SET `equipment_id` = '0' WHERE `entry` = '26259';

UPDATE `creature`, `creature_template` SET `creature`.`curhealth`=`creature_template`.`minhealth`,`creature`.`curmana`=`creature_template`.`minmana` WHERE `creature`.`id`=`creature_template`.`entry`;

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 29.07.2015 11:52:36 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079/#comment-20547073

<hr>

any news?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 10.09.2015 15:49:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1079/#comment-21679639

<hr>

[Core/DB/Cleanup] Cleanup startup errors.

```
              Closes #1079
              Thanks to PadreWoW.
```

→ <<cset bd25586ed91f>>
